### PR TITLE
ENT-2951: Added alert for missing subscription

### PR DESCRIPTION
**Pull request by irfanuddinahmad**
JIRA: https://openedx.atlassian.net/browse/ENT-2951

image

### CODE REVIEW

File: src/components/subscriptions/SubscriptionManagementPage.jsx [hunk 1]
```diff
@@ -45,75 +45,77 @@ function SubscriptionManagementPage({ enterpriseSlug }) {
...
-                        </div>
-                        <div className="col-12 col-lg-7">
-                          <AddUsersButton
-                            onSuccess={() => setStatus({
-                              visible: true,
-                              alertType: 'success',
-                              message: 'Successfully assigned license(s)',
-                            })}
-                          />
+            <div className="row mb-3">
```
[the remark this answers is not in the document]
Comment (sameenfatima78) [reply src/components/subscriptions/SubscriptionManagementPage.jsx:64]: Also there are some conflicts that need to be resolved first.

**Comment by muhammad-ammar** [comment]
@irfanuddinahmad Can you rebase your branch?

**Comment by irfanuddinahmad** [comment]
> @irfanuddinahmad Can you rebase your branch?

@muhammad-ammar Done!

**Comment by irfanuddinahmad** [comment]
> The status alert appears for a short while when we open Subscription management page of an enterprise with an active subscription. It appears and then disappears instantly.

Fixed

File: src/components/subscriptions/SubscriptionData.jsx [hunk 2]
```diff
@@ -81,6 +83,17 @@ export default function SubscriptionData({ children }) {
...
+  if (!isLoading) {
+    return (
+      <StatusAlert
+        className="mt-3"
+        alertType="danger"
+        message={`Your organization does not have any active subscriptions to manage.
+        If you believe you are seeing this message in error,
+        please reach out to the edX Customer Success team at customersuccess@edx.org.`}
+      />
+    );
```
Comment (muhammad-ammar) [inline src/components/subscriptions/SubscriptionData.jsx:95]: @irfanuddinahmad  As per my understanding, we need to make this decision on `SubscriptionManagementPage` and either display the  usual UI or error message.

**Comment by irfanuddinahmad** [comment]
> The status alert appears for a short while when we open Subscription management page of an enterprise with an active subscription. It appears and then disappears instantly.

This does not seem to have any straightforward solution.

**Comment by zamanafzal** [comment]
> The status alert appears for a short while when we open Subscription management page of an enterprise with an active subscription. It appears and then disappears instantly.

I think we can file a new ticket for this issue and resolve that on the basis of the priority it has.

**Comment by sameenfatima78** [comment]
@irfanuddinahmad I agree with Zaman

**Pull request merged**